api-testing/multicloud-api/gradlew

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env sh
```

Comment:
Are we taking this code as-is from anywhere?

Reply:
It's auto-generated file during creating gradle project this will be help through cmd line build the code.

Reply:
do we need to keep the auto generated code to github? Will it not get generated during the build process, if we give the steps to build?

Reply:
It is okay, These are auto generated files

Reply:
ok